src/cascadia/WinRTUtils/inc/Utils.h

```diff
@@ -17,6 +17,9 @@ winrt::Windows::Foundation::IAsyncOperation<winrt::hstring> FilePicker(HWND pare
     DWORD flags{};
     THROW_IF_FAILED(fileDialog->GetOptions(&flags));
     THROW_IF_FAILED(fileDialog->SetOptions(flags | FOS_FORCEFILESYSTEM | FOS_NOCHANGEDIR | FOS_DONTADDTORECENT)); // filesystem objects only; no recent places
+
+    // The MSVC 14.31.31103 toolset seems to misdiagnose this line.

```

Comment:
1: spell fix the word toolset

2: we should follow the STL team's convention and use "BODGY" to indicate compiler workarounds maybe?